tests/unit/browser/webengine/test_spell.py

```diff
@@ -0,0 +1,40 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
```

Comment:
You can now add the `spell.py` file and this to `PERFECT_FILES` in `scripts/dev/check_coverage.py`.